### PR TITLE
Changed access methods

### PR DESCRIPTION
**Pull request by nawarani**
Leaderboard structure was changed so that there is direct access to the class to add a player when they win and print updated version of leaderboard.

### CODE REVIEW

File: src/main/java/LeaderBoard.java [hunk 1]
```diff
@@ -72,8 +41,30 @@ private static void writeData(ArrayList<String[]> leaderboard) {
         }
     }
 
+    //write file v2
+    public static void addPlayer(String player_name) {
```
Comment (AscendedWise) [inline src/main/java/LeaderBoard.java:45]: Thank you this makes adding and fetching saved data much easier for the leaderboardUI

**Review by AscendedWise — APPROVED** [review]
I approve of these changes

**Review by AscendedWise — APPROVED** [review]
Thank you for adding a method to sort the leaderboard, I approve again

File: src/main/java/LeaderBoard.java [hunk 2]
```diff
@@ -1,57 +1,49 @@
...
+        private String name;
+        private int wins;
+        protected Player(String name, int wins){
+            this.name = name;
+            this.wins = wins;
         }
-        if (!(existing_player)) {
-            String[] new_player = {input, "1"};
-            leaderboard.add(new_player);
+        // Good coding practice bla bla getter functions
```
Comment (123Chan) [inline src/main/java/LeaderBoard.java:22]: Nice, I think this makes code more readable as well.

**Review by 123Chan — APPROVED** [review]

**Pull request merged**